pulp_ansible/app/tasks/utils.py

```diff
@@ -13,8 +13,13 @@ def get_page_url(url, page=1):
     """Get URL page."""
     parsed_url = urlparse(url)
     new_query = parse_qs(parsed_url.query)
-    new_query["page"] = page
-    new_query["page_size"] = PAGE_SIZE
+
+    if "v1" in url or "v2" in url:
```

Comment:
Can the sync code deduplicate this check with the other one @daviddavis linked to? That way all the checking is done on a variable that checks the url once.

Reply:
I don't know what is the check you are talking about

Reply:
Here you go:

https://github.com/pulp/pulp_ansible/blob/8e4168dca96ab00d329ef8f5e0e9886218ef1302/pulp_ansible/app/tasks/collections.py#L363-L364